compute/admin_guide/install/install-gke.adoc

```diff
@@ -44,7 +45,7 @@ $ kubectl create clusterrolebinding your-dev-user-cluster-admin-binding \
     --user=your-dev-user@your-org.iam.gserviceaccount.com
 ----
 
-. With the *Kubernetes Engine Developer* service account, continue with the standard Kubernetes install procedure for Prisma Cloud Console and Defenders, <<_install_console,starting here>>.
+. With the *Kubernetes Engine Developer* service account, continue with the standard Kubernetes install procedure for the xref:install_kubernetes.adoc#install_console[Prisma Cloud Console] and xref:./install_defender/install_cluster_container_defender.adoc#deamonset-gke[Defenders],
```

Comment:
Hi @rcaballeromx this doc doesn't have all the changes as well.

Reply:
@MorShabiPAN Did you make changes to other files or is this the last one?

Reply:
All changed files are listed in the PR. I think this is it.